_layouts/default.html

```diff
@@ -0,0 +1,157 @@
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>dxw Playbook</title>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport"
+    content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="//unpkg.com/docsify/lib/themes/vue.css">
+  <link rel="stylesheet" href="/build/assets/css/styles.css">
+  <style>
+    :root {
+      --theme-color: #243746;
+    }
+  </style>
+  <style>
```

Comment:
Can we pull these into stylesheets and run autoprefixer on them instead of doing manual prefixing?